style/comman.css

```diff
@@ -0,0 +1,66 @@
+*{
+    margin: 0px;
+    padding: 0px;
+    border-radius:var(--border-radius) ;
+    text-transform: capitalize;
+    /* background-color: var(--lightmode-bg-color); */
+    text-decoration: none;
+    list-style: none;
+    box-sizing: border-box;
+}
+.flex{
```

Comment:
you can correct the name "common"

text-transform can be used only on heading.

you can use smallcases in code snippets